infra/aws/terraform/cdn.packages.k8s.io/terraform.tfvars

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+region = "eu-central-1"
```

Comment:
We could use this variable to replace some hard-coded uses.

Reply:
We only use hard-coded value in `providers.tf` for which Terraform requires the value to be hard-coded.

Reply:
I think you can use a variable there.
```hcl
provider "aws" {
  region  = var.aws_region
}
```
this kind of thing

Reply:
I already use it for that: https://github.com/kubernetes/k8s.io/pull/5536/files#diff-c01dbdb35273cc39e27cbfad6d11f4a34147bf59438edde1ffd7d42a977dbb46R36-R43

I only don't use in the `backend` block because it's not allowed by Terraform. `us-east-1` is also hard-coded, but defining it as a variable doesn't make sense because certificates must be placed in that region.

Reply:
Ah right. Well, the state isn't going to move region either.